index.html

```diff
@@ -269,9 +269,9 @@ <h3>Subscriptions</h3>
                     <div class="feature-description">
                         <p><strong>Never miss updates again:</strong></p>
                         <ul>
-                            <li>Subscribe a channel with one click</li>
-                            <li>Use your custom feed</li>
-                            <li>Watch your favourite artists</li>
+                            <li>Subscribe to a channel with one click</li>
+                            <li>Use your customized feed</li>
+                            <li>Watch your favourite channels</li>
```

Comment:
Makes sense in the YouTube context, but might not necessarily make sense for other services we support. Maybe we can find a more "media agnostic" term?

Reply:
uploader? :D

Why exactly is that there anyway, under the header of subscriptions?

Reply:
You're not watching the uploader, you're watching their uploaded stuff. But you may also just listen, not watch, etc... That's kind of the issue.

> Why exactly is that there anyway, under the header of subscriptions? Not really related to that feature, I can just watch my favourite artists anywhere else in the app, right?

CC @TobiGr

Reply:
`content feed` is a pretty technical but IMO the most precise term. But we address end users as well with the website. So that's not necessarily a good wording.

Let's keep it "channels".